miden-tx/src/errors/mod.rs

```diff
@@ -8,7 +8,7 @@ use miden_objects::{
 use miden_verifier::VerificationError;
 use vm_processor::ExecutionError;
 
-pub mod tx_kernel_errors;
+pub mod vm_errors;
```

Comment:
I would probably change this back to `tx_kernel_errors`. The reason for this is that in the future we'll also have block kernel errors and batch kernel errors.

I agree that putting note scripts into kernel errors is not ideal - but maybe that's fine for now. Let's create an issue thought figure out how to deal with errors that can come from multiple sources.

Reply:
Makes sense, I renamed them back :+1: 

Not sure I captured what you meant exactly, but here is the issue: https://github.com/0xPolygonMiden/miden-base/issues/910